src/main/java/org/dependencytrack/persistence/VulnerabilityQueryManager.java

```diff
@@ -887,4 +888,49 @@ public void deleteAffectedVersionAttributions(final Vulnerability vulnerability)
         query.setParameters(vulnerability);
         query.deletePersistentAll();
     }
+
+    /**
+     * Binds the two objects together in a corresponding join table.
+     *
+     * @param vulnerability a Vulnerability object
+     * @param tags    a List of Tag objects
+     */
+    @SuppressWarnings("unchecked")
+    public void bind(Vulnerability vulnerability, List<Tag> tags) {
+        final Query<Tag> query = pm.newQuery(Tag.class, "vulnerabilities.contains(:vulnerability)");
+        final List<Tag> currentVulnerabilityTags = (List<Tag>) query.execute(vulnerability);
+        pm.currentTransaction().begin();
```

Comment:
Consider using `runInTransaction` instead, as it will handle rollbacks in case something breaks between `begin` and `commit`.